src/Facets/EcoFacet.sol

```diff
@@ -0,0 +1,256 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+
+pragma solidity ^0.8.17;
+
+import { ILiFi } from "../Interfaces/ILiFi.sol";
+import { IEcoPortal } from "../Interfaces/IEcoPortal.sol";
+import { LibAsset } from "../Libraries/LibAsset.sol";
+import { LibSwap } from "../Libraries/LibSwap.sol";
+import { ReentrancyGuard } from "../Helpers/ReentrancyGuard.sol";
+import { SwapperV2 } from "../Helpers/SwapperV2.sol";
+import { Validatable } from "../Helpers/Validatable.sol";
+import { LiFiData } from "../Helpers/LiFiData.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { InvalidConfig, InvalidReceiver, InformationMismatch } from "../Errors/GenericErrors.sol";
+
+/// @title EcoFacet
+/// @author LI.FI (https://li.fi)
+/// @notice Provides functionality for bridging through Eco Protocol
+/// @custom:version 1.0.0
+contract EcoFacet is ILiFi, ReentrancyGuard, SwapperV2, Validatable, LiFiData {
+    /// Storage ///
+
+    // solhint-disable-next-line immutable-vars-naming
```

Comment:
pls remove the solhint disable comment and rename the variable to PORTAL to be consistent with our other (more recent) facets

Reply:
fixed: bd753870